refarch-integrations/refarch-dms-integration/refarch-dms-integration-core/src/test/java/de/muenchen/refarch/integration/dms/adapter/out/s3/S3AdapterTest.java

```diff
@@ -63,47 +72,19 @@ void setup() {
     void testLoadFileFromFilePath()
             throws IOException, DocumentStorageException, DocumentStorageClientErrorException, DocumentStorageServerErrorException {
 
-        final String pdfPath = "files/test/test-pdf.pdf";
-        final String pngPath = "files/test/test-png.png";
+        final List<String> filePaths = List.of(PDF_PATH, PNG_PATH);
 
-        final List<String> filePaths = List.of(pdfPath, pngPath);
+        final byte[] testPdf = new ClassPathResource(PDF_PATH).getInputStream().readAllBytes();
+        final byte[] testPng = new ClassPathResource(PNG_PATH).getInputStream().readAllBytes();
 
-        final byte[] testPdf = new ClassPathResource(pdfPath).getInputStream().readAllBytes();
-        final byte[] testPng = new ClassPathResource(pngPath).getInputStream().readAllBytes();
-
-        when(documentStorageFileRepository.getFile(pdfPath, 3)).thenReturn(testPdf);
-        when(documentStorageFileRepository.getFile(pngPath, 3)).thenReturn(testPng);
+        when(documentStorageFileRepository.getFile(PDF_PATH, 3)).thenReturn(testPdf);
+        when(documentStorageFileRepository.getFile(PNG_PATH, 3)).thenReturn(testPng);
         when(documentStorageFileRepository.getFileSize(anyString())).thenReturn(1_000_000L);
 
         final List<Content> contents = this.s3Adapter.loadFiles(filePaths);
 
-        final Content pdfContent = new Content("pdf", "test-pdf", testPdf);
-        final Content pngContent = new Content("png", "test-png", testPng);
-
-        assertTrue(contents.contains(pdfContent));
-        assertTrue(contents.contains(pngContent));
-    }
-
-    @Test
-    void testLoadFileFromFilePathWithStorageUrl()
```

Comment:
Did you remove some test cases in this class? Why is that?

Reply:
Completely removed the storage url overwrite as it was DigiWF specific, and the test than was a duplicate.